app/src/main/res/values/strings.xml

```diff
@@ -232,13 +232,13 @@
     <string name="unsupport_magisk_msg">This version of the app does not support Magisk version lower than %1$s.\n\nThe app will behave as if no Magisk is installed, please upgrade Magisk as soon as possible.</string>
     <string name="unsupport_general_title">Abnormal State</string>
     <string name="unsupport_system_app_msg">Running this app as a system app is not supported. Please revert the app to a user app.</string>
-    <string name="unsupport_other_su_msg">An \"su\" command that does not belong to Magisk is detected. Please remove the other unsupported su.</string>
+    <string name="unsupport_other_su_msg">Please remove the program that sent a \"su\" command and handle all requests through Magisk.</string>
```

Comment:
To be honest, the new string looks more confusing to me. Isn't this message shown when another su binary (e.g. SuperSU) is installed?
More specifically, it's the part _"the program that sent a \"su\" command"_ that confuses me. All apps that use root permissions "send" `su` commands.

Reply:
```suggestion
    <string name="unsupport_other_su_msg">Please remove that just tried to handle a \"su\" command and handle all requests through Magisk.</string>
```

Reply:
You probably meant to say "...remove **the app** that just..." :)
In any case, the message worded as it is now makes sense only if the other root app is detected when an app tries to access su.
If this check instead happens as soon as Magisk is installed, it would be misleading to say that the other su app _just tried_ to handle a su command.